docs/_guides/apamajms.md

```diff
@@ -0,0 +1,548 @@
+(config-client-username)# acl-profile default	
+(config-client-username)# client-profile default
+(config-client-username)# no shutdown
+(config-client-username)# exit
+(config)#
+```
+
+#### Setting up Guaranteed Messaging Endpoints
+This integration guide shows receiving messages within the Apama from a single JMS Queue. For illustration purposes, this queue is chosen to be an exclusive queue with a message spool quota of 2GB matching quota associated with the message VPN. The queue name chosen is “apama”.
+
+```
+(config)# message-spool message-vpn apama
+(config-message-spool)# create queue apama
+(config-message-spool-queue)# access-type exclusive
+(config-message-spool-queue)# max-spool-usage 2000
+(config-message-spool-queue)# permission all delete
+(config-message-spool-queue)# no shutdown
+(config-message-spool-queue)# exit
+(config-message-spool)# exit
+(config)#
+```
+
+#### Setting up Solace JNDI References
+
+To enable the JMS clients to connect and look up the Queue destination required by Apama, there are two JNDI objects required on the Solace Message Router:
+
+* A connection factory: jms/cf/apama
+* A queue destination: JNDI/Q/apama
+
+They are configured as follows:
+
+```
+(config)# jndi message-vpn apama
+(config-jndi)# create connection-factory jms/cf/apama
+(config-jndi-connection-factory)# property-list messaging-properties
+(config-jndi-connection-factory-pl)# property default-delivery-mode persistent
+(config-jndi-connection-factory-pl)# exit
+(config-jndi-connection-factory)# property-list transport-properties
+(config-jndi-connection-factory-pl)# property direct-transport false
+(config-jndi-connection-factory-pl)# property "reconnect-retry-wait" "3000"
+(config-jndi-connection-factory-pl)# property "reconnect-retries" "20"
+(config-jndi-connection-factory-pl)# property "connect-retries-per-host" "5"
+(config-jndi-connection-factory-pl)# property "connect-retries" "1"
+(config-jndi-connection-factory-pl)# exit
+(config-jndi-connection-factory)# exit
+(config-jndi)#
+(config-jndi)# no shutdown
+(config-jndi)# exit
+(config)#
+```
+
+###	Step 3 – Obtain Apama Streaming Analytics
+
+The Apama Streaming Analytics can be obtained one of 2 ways.     
+1.	If you are in an organization that is an existing Apama customer, it is likely your organization already has Apama Streaming Analytics and corporate policies about their use.  You will have to contact your application management team in regards to a Apama Streaming Analytics license.
+2.	If you are new to Apama Streaming Analytics, you can obtain Apama Community Edition from the [Apama Community Edition Downloads](http://www.apamacommunity.com/downloads/){:target="_blank"}. For help getting started with your Apama Streaming Analytics Community Edition you can refer to [Apama Community Edition Docs](http://www.apamacommunity.com/docs/){:target="_blank"}.
+
+### Step 4 – Coding Solace message producer
+
+**Solace JCSMP project**
```

Comment:
Why is this example not JMS? So far JCSMP hasn't been mentioned up to this point. I feel this should be a JMS producer given the above configuration and discussion. I realize this is simply a sample. Ideally however, you'd use only one Solace API for simplicity.